single.sh

```diff
@@ -19,6 +19,13 @@ fi
 
 nakedname=$(echo "${CFG}" | rev | cut -d, -f1 | cut -d'/' -f 1 | cut -d'.' -f 2-99 | rev)
 
+if [ -n "${TRANSLATION}" ]; then
+  python3 translate.py ${TRANSLATION}
+  export LATEX_CMD="xelatex"
+else
+  export LATEX_CMD="pdflatex"
```

Comment:
If xelatex works better, maybe simply always use xelatex instead?

Reply:
I agree - it looks like pdflatex is quite obsolated, and xelatex can handle all translation with Unicode fonts perfectly.

Reply:
Sure, I'll change it to xelatex permanently then